src/components/donation/Steps/DonateMethods/DonateMethods.tsx

```diff
@@ -37,14 +39,14 @@ const methods: {
     panel: Crypto,
   },
 };
-const allMethodIds: DonateMethodId[] = ["stripe", "stocks", "daf", "crypto"];
+const allMethodIds: DonateMethodId[] = ["stripe", "daf", "stocks", "crypto"];
 
 const tabClasses = (selected: boolean) =>
   `${
     selected
       ? "font-medium bg-[--accent-secondary] text-navy-d4"
-      : "border border-gray-l4 @md/steps:border-none text-navy-l1"
-  }  flex items-center gap-2 p-2 @md/steps:px-3 @md/steps:py-[1.15rem] @md/steps:grid @md/steps:grid-cols-subgrid @md/steps:col-span-2 focus:outline-none @md/steps:w-28 rounded @md/steps:rounded-none`;
+      : "border border-gray-l4 @xl/steps:border-none text-navy-l1"
```

Comment:
```suggestion
      : "border border-gray-l4 @md/steps:border-none text-navy-l1"
```

increase mobile breakpoint so that spelled out `DAF` text doens't shrink tab panel too much on smaller screens